.heroku/vendor/pygobstones-lang

```diff
@@ -1 +1 @@
-Subproject commit 141d657e1317132a215f54cec952580032d86f6d
+Subproject commit 34e62b00bf7e0ce23094be7c9a8cc6237c6ebdde
```

Comment:
Actually I didn't update it to the newest version, because there is a problem on that one: https://github.com/gobstones/PyGobstones-Lang/issues/15

I pull the same version which is now on stones-spec, and concerning the dependency problem I've already made an issue to discuss it: #61